message_center_compassion/mappings/field_to_json.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Flückiger Nathan <nathan.fluckiger@hotmail.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields
+
+
+class FieldToJson(models.Model):
+
+    _name = "field_to_json"
+
+    field_name = fields.Char(string="Odoo Field Name")
+    json_name = fields.Char(string="Json Field Name")
```

Comment:
You could add as well the relational Many2Many field here for better handling in code.
```
mapping_ids = fields.Many2many('compassion.mapping', 'Associated mappings')
```